src/quartz/schemas/OrganizationWithToken.yml

```
@@ -6,6 +6,10 @@ allOf:
         description: Authentication token to manage the organization and its resources in IDPE
         readOnly: true
         type: string
+      userName:
```

```suggestion
      name:
```

The OEM API knows nothing about users.